src/service/resources-impl.js

```diff
@@ -286,89 +292,80 @@ export class ResourcesImpl {
 
   /**
    * @param {!Array<!IntersectionObserverEntry>} entries
-   * @param {!IntersectionObserver} unusedObserver
    * @private
    */
-  intersects_(entries, unusedObserver) {
+  intersects_(entries) {
+    // TODO(willchou): Remove assert once #27167 is fixed.
+    devAssert(this.prerenderSize_ == 1);
```

Comment:
What's the value of the `devAssert()` for this?

Reply:
Mostly as an FYI for readers. Assert would trigger for the few people that are playing with `prerenderSize` in development (maybe just me).